Use of tainted arguments in exec is deprecated at /var/www/cgi-bin/cvsweb.cgi line 2043.
===================================================================
RCS file: /home2/webcvs/mirror/kdepim/libkdenetwork/kmime_codec_qp.h,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- kdepim/libkdenetwork/kmime_codec_qp.h	2003/07/26 14:51:52	1.4
+++ kdepim/libkdenetwork/kmime_codec_qp.h	2003/10/30 14:15:30	1.5
@@ -54,19 +54,11 @@ public:
     int result = 3*insize;
     // then after 25 hexchars comes a soft linebreak: =(\r)\n
     result += (withCRLF ? 3 : 2) * (insize/25);
-
+    
     return result;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    // all chars unencoded:
-    int result = insize;
-    // but maybe all of them are \n and we need to make them \r\n :-o
-    if ( withCRLF )
-      result += insize;
-
-    return result;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
@@ -92,11 +84,7 @@ public:
     return 3*insize;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    (void)withCRLF; // keep compiler happy
-    // equally simple: nothing is encoded at all, so:
-    return insize;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
@@ -121,11 +109,7 @@ public:
     return 3*insize;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    (void)withCRLF; // keep compiler happy
-    // same as for "q" encoding:
-    return insize;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
